### PR TITLE
add eessi-upload-to-staging script to build container

### PR DESCRIPTION
**Pull request by boegel**
~draft PR, not tested yet...~

This makes uploading tarball of additional software installations a bit easier, no need to copy stuff around first...

### CODE REVIEW

File: containers/Dockerfile.EESSI-build-node-debian10 [hunk 1]
```diff
@@ -25,3 +25,13 @@ RUN echo 'CVMFS_QUOTA_LIMIT=10000' > /etc/cvmfs/default.local \
 RUN mkdir -p /cvmfs/pilot.eessi-hpc.org
 
 RUN useradd -ms /bin/bash eessi
+
+# see https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html#cliv2-linux-install
+RUN curl https://awscli.amazonaws.com/awscli-exe-linux-$(uname -m)-2.2.35.zip -o awscliv2.zip \
```
Comment (terjekv) [inline containers/Dockerfile.EESSI-build-node-debian10:30]: Should we avoid hard coding the version number?

**Review by terjekv — APPROVED** [review]

File: containers/Dockerfile.EESSI-build-node-debian10 (hunk 1 above)
Comment (boegel) [reply containers/Dockerfile.EESSI-build-node-debian10:30]: I actually prefer hardcoding it, to have control over what we install exactly...

But I'm happy to drop this, and always install latest instead.

Comment (terjekv) [reply containers/Dockerfile.EESSI-build-node-debian10:30]: Ah, no, I ment we should store it in a variable or similar. We should certainly not just grab latest.

**Review by bedroge — DISMISSED** [review]

**Review by bedroge — DISMISSED** [review]

**Comment by bedroge** [comment]
@boegel the check is failing on `ppc64le` because of:
```
 > [stage-1 10/11] RUN curl https://awscli.amazonaws.com/awscli-exe-linux-$(uname -m)-2.2.35.zip -o awscliv2.zip   && unzip awscliv2.zip   && ./aws/install   && rm -r aws:

  0  5795    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  5795  100  5795    0     0   9198      0 --:--:-- --:--:-- --:--:--  9392
#15 0.929 Archive:  awscliv2.zip
#15 0.930   End-of-central-directory signature not found.  Either this file is not
#15 0.930   a zipfile, or it constitutes one disk of a multi-part archive.  In the
#15 0.930   latter case the central directory and zipfile comment will be found on
#15 0.930   the last disk(s) of this archive.
#15 0.931 unzip:  cannot find zipfile directory in one of awscliv2.zip or
#15 0.931         awscliv2.zip.zip, and cannot find awscliv2.zip.ZIP, period.
```

**Comment by terjekv** [comment]
ppc64le (or POWER at all) is not a supported platform for awscli: https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html#cliv2-linux-install

Thankfully AWS wants aarch64 for arm, so that should work.

**Comment by bedroge** [comment]
The pip package does seem to work on `ppc64le`, so maybe we could use do `pip install awscli` instead?

**Comment by boegel** [comment]
> The pip package does seem to work on `ppc64le`, so maybe we could use do `pip install awscli` instead?

Yes, and there's actually other good reasons to stick to `awscli` 1.x (which is available through PyPI, 2.x is not), grab a ☕  and go read https://github.com/aws/aws-cli/issues/4947...

**Review by bedroge — APPROVED** [review]
lgtm

**Pull request merged**